src/index.tsx

```diff
@@ -6,13 +6,17 @@ import { createRoot } from "react-dom/client";
 import "./styles/styles.less";
 import FastlegeContainer from "./containers/FastlegeContainer";
 import { minutesToMillis } from "@/utils/timeUtils";
-import { QueryClient, QueryClientProvider } from "react-query";
-import { ReactQueryDevtools } from "react-query/devtools";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+import { ReactQueryDevtools } from "@tanstack/react-query-devtools";
 import { isClientError } from "@/api/errors";
 
 const queryClient = new QueryClient({
   defaultOptions: {
+    mutations: {
```

Comment:
Tilsvarer v3 (https://tanstack.com/query/v4/docs/react/guides/migrating-to-react-query-4#queries-and-mutations-per-default-need-network-connection-to-run)